lib/msal-node/tsconfig.json

```diff
@@ -10,9 +10,9 @@
     "rootDir": "./",
     "strict": true,
     "noImplicitAny": true,
-    "strictNullChecks": true,
+    "strictNullChecks": false,
```

Comment:
The sooner this is set to `true`, the better.

Reply:
Agreed. I'll probably tackle this one next.